src/main/java/com/coinbase/examples/Main.java

```diff
@@ -109,6 +109,28 @@ public static void main(String[] args) {
                                 .build());
             }
 
+            // Create and Accept RFQ
```

Comment:
Move this to another file? I would make additional main methods for the key examples you want to show. This currently makes too many things happen to run without commenting out major sections